app/views/categories/index.html.erb

```diff
@@ -1,39 +1,69 @@
 <%= provide(:title, "index") %>
-<div class="text-center">
-    <h1>Lessons</h1>
-</div>
-<div class="row d-flex justify-content-center mt-4">
-    <%= will_paginate @category%>
-</div>
+<div class="row mt-5 mx-auto justify-content-center">
+  <div class="card" style="width: 10rem; height: 10rem">
+    <div class="card-header">
+        <%= link_to "Dashboard", categories_path %>
+    </div>
+          <ul class="list-group list-group-flush">
+              <li class="list-group-item">
+                  <%= link_to "Categories", categories_path %>
+              </li>
+              <li class="list-group-item">
+                  <%= link_to "Users" %>
+              </li>
+              <li class="list-group-item text-center">
+                  <%= link_to "Back", :back, class: "btn btn-outline-info w-10" %>
+              </li>
+          </ul>
+  </div>
 
-<div class="container">
-<table class="table table-striped">
-  <thead>
-    <tr>
-      <th scope="col">#</th>
-      <th scope="col">Title</th>
-      <th scope="col">Description</th>
-      <th colspan="3"></th>
-    </tr>
-  </thead>
-
-  <tbody>
-      <% @category.each do |category| %>
-        <tr>
-          <td><%= category.id %></td>
-          <td><%= link_to category.title %></td>
-          <td><%= category.description %></td>
-        <% if logged_in? && current_user.is_admin? %> 
-          <td><%= link_to raw("<i class='fas fa-pen text-success'></i>"), edit_category_path(category) %></td>
-          <td><%= link_to raw("<i class='fa fa-trash text-danger'></i>"), category, method: :delete, data: { confirm: "You sure?" } %></td>
-        <% end %>
-        </tr>
-      <% end %>
-  </tbody>
-</table>
+  <div class="ml-5">
+      <div class="card text-primary bg-light" style="width: 60rem;">
+          <div class="row card-header">
+            <h3 class="offset-1">Lessons</h3>
+            <%= link_to "Create new category", new_category_path, class: "btn btn-outline-danger offset-md-7" %>
+          </div>
+          <div class="card-body">
+              <div class="container">
+                  <table class="table table-striped">
+                      <thead>
+                          <tr>
+                              <th scope="col">Title</th>
+                              <th scope="col">Description</th>
+                              <th colspan="3"></th>
+                          </tr>
+                      </thead>
+                      <tbody>
+                        <% @category.each do |category| %>
+                            <tr>
+                                <td><%= link_to category.title, words_path(category) %></td>
```

Comment:
update this code to :`<td><%= link_to category.title, category_words_path(category) %></td>`